source/documentation/deploying_services/mysql.md

```diff
@@ -132,7 +132,7 @@ To create a service and bind it to your app:
 
 ### Accessing MySQL from your app
 
-Your app must make a [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) connection to the MySQL service. Most libraries use TLS by default.
+Your app must make a [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) connection to the MySQL service. Some libraries use TLS by default, but others will need to be explicitly configured.
```

Comment:
Probably not inline examples (too much noise), but linking to [example apps](https://github.com/alphagov/paas-cf/tree/master/platform-tests/example-apps)? Which presumably give correctly configured examples.

Possibly this could come later once their are more example combinations to refer to.